### PR TITLE
bump lock file for 7.6.1

### PR DESCRIPTION
**Pull request by logstashmachine**

### CODE REVIEW

**Comment by jsvd** [comment]
jenkins test this please

File: Gemfile.jruby-2.5.lock.release [hunk 1]
```diff
@@ -39,28 +39,28 @@ GEM
...
+    aws-sdk-resources (2.11.451)
+      aws-sdk-core (= 2.11.451)
     aws-sdk-v1 (1.67.0)
       json (~> 1.4)
       nokogiri (~> 1)
     aws-sigv4 (1.1.0)
       aws-eventstream (~> 1.0, >= 1.0.2)
     back_pressure (1.0.0)
-    backports (3.15.0)
+    backports (3.16.1)
```
Comment (colinsurprenant) [inline Gemfile.jruby-2.5.lock.release]: minor bump here?

File: Gemfile.jruby-2.5.lock.release [hunk 2]
```diff
@@ -543,17 +543,17 @@ GEM
...
     murmurhash3 (0.1.6-java)
     mustache (0.99.8)
     mustermann (1.0.3)
     naught (1.1.0)
     nio4r (2.5.2-java)
-    nokogiri (1.10.7-java)
+    nokogiri (1.10.8-java)
     numerizer (0.1.1)
-    octokit (4.15.0)
+    octokit (4.16.0)
```
Comment (colinsurprenant) [inline Gemfile.jruby-2.5.lock.release]: also minor bump?

Comment (colinsurprenant) [inline Gemfile.jruby-2.5.lock.release]: also minor bump?

File: Gemfile.jruby-2.5.lock.release [hunk 3]
```diff
@@ -573,7 +573,7 @@ GEM
     public_suffix (3.1.1)
     puma (4.3.1-java)
       nio4r (~> 2.0)
-    rack (2.1.1)
+    rack (2.2.2)
```
Comment (colinsurprenant) [inline Gemfile.jruby-2.5.lock.release]: minor bump?

File: Gemfile.jruby-2.5.lock.release [hunk 4]
```diff
@@ -605,7 +605,7 @@ GEM
       faraday (> 0.8, < 2.0)
     semantic_logger (3.4.1)
       concurrent-ruby (~> 1.0)
-    sequel (5.28.0)
+    sequel (5.29.0)
```
Comment (colinsurprenant) [inline Gemfile.jruby-2.5.lock.release]: minor bump?

**Comment by colinsurprenant** [comment]
Other than the few minor bumps commented in-line, LGTM.

**Pull request merged**